app/posts/apply-for-teacher-training/2023-06-19-moving-equality-diversity-survey.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Moving the equality and diversity survey
+description: We moved the equality and diversity survey to a new section on the application form.
+date: 2023-06-19
+tags:
+  - applications
+  - equality and diversity
+ogImage:
+  src: /apply-for-teacher-training/moving-equality-diversity-survey/equality-survey-cover-image.png
```

Comment:
```suggestion
  src: equality-survey-cover-image.png
```
You no longer need the file path for the OpenGraph image, as the template handles it for you.